app/models/user.rb

```diff
@@ -493,8 +494,8 @@ def watching_location(nwlat, selat, nwlng, selng)
     end
   end
 
-  def recent_locations(limit = 5)
-    recent_nodes = self.nodes.includes(:tag)
+  def recent_locations(_limit = 5)
```

Comment:
In this case, you have the opportunity to use your argument:

```rb
...
.order(created: :desc)
.limit(limit)
```

Reply:
Done!